ext/session/tests/user_session_module/session_set_save_handler_error5.phpt

```diff
@@ -0,0 +1,19 @@
+--TEST--
+Test session_set_save_handler() function : error functionality
+--EXTENSIONS--
+session
+--INI--
+session.save_handler=
+--FILE--
+<?php
+try {
+    session_set_save_handler(new \SessionHandler(), true);
+    session_start();
+} catch (Throwable $e) {
+    echo $e->getMessage() . "\n";
```

Comment:
Personally I prefer seeing what exception is being thrown 
```suggestion
    echo $e::class . ': ' . $e->getMessage() . "\n";
```

Reply:
Make sense. Added

Reply:
Did you forget to push?

Reply:
Yes, sorry. Internet connection interruption.